datarepo-clienttests/src/main/resources/configs/BulkLoadScale.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "BulkLoadScale",
+  "description": "One user ingests many files into one dataset",
+  "serverSpecificationFile": "localhost.json",
+  "billingAccount": "00708C-45D19D-27AAFA",
+  "kubernetes": {
+    "numberOfInitialPods" : 1
+  },
+  "application": {
+    "maxStairwayThreads" : 110,
+    "loadConcurrentFiles": 20
```

Comment:
nit: rename to `BulkLoadScale100_20.json` to be consistent with other files?

Reply:
We could get rid of that file. It was the first one i made that then grew into the others.

Reply:
removed!